docs/modules/md5.md

```diff
@@ -0,0 +1,11 @@
+# md5
+
```

Comment:
can you add a small description? just mention that you should use a more secure algorithm if possible, but otherwise can use built-in functionality

Reply:
done